landingzones/lz-machinelearning/lz.bicep

```
@@ -342,7 +349,7 @@ module databricks '../../azresources/analytics/databricks/main.bicep' = {
   }
 }
 
-module aksCluster '../../azresources/containers/aks/main.bicep' = {
+module aksCluster '../../azresources/containers/aks/main.bicep' = if (aks.enabled) {
   name: 'deploy-aks-${aks.networkPlugin}'
```

The name of the deployment will be evaluated even through aks.enabled is set to false.  Therefore we should change the name to have a ternary boolean check.

Can this be replaced as

```suggestion
  name: 'deploy-aks-${aks.enabled ? aks.networkPlugin : ''}'
```